doc/source/getting_started/index.rst

```diff
@@ -64,6 +64,8 @@ Install DPF Server
 
         ANSYS_DPF_ACCEPT_LA=Y
 
+For more information about DPF licensing, see :ref:`_ref_licensing`.
```

Comment:
This link is not working. I donwloaded the doc artifact form the CI to read the final render of the doc, and this is how it looks like:
![image](https://github.com/ansys/pydpf-core/assets/107186344/85c1e914-1145-4c7c-86ae-b941fdacbb01)

Reply:
Fixed